server/models/Fund.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+
+const FundSchema = mongoose.Schema({
+    user: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'user'
+    },
+    name: {
+        type: String,
+        required: true
+    },
+    description: {
+        type: String,
+        required: true
+    },
+    goal: {
+        type: String,
+        required: true
+    },
+    deadline: {
+        type: Date,
+        default: Date.now
+    },
+    url: {
+        type: String
+    }
+});
+
+const Fund = mongoose.model('fund', FundSchema);
```

Comment:
looks like this is dependant on Fund model branch. this is okay but perhaps make a note of this so we don't re-review this code :)

Reply:
Sure I have added comment on this.